### PR TITLE
Sign in sing up margins

### LINKED ISSUE #138: Sign up / sign up page
Make margins equal

Screen Shot 2020-11-20 at 20 22 50

### PR DESCRIPTION
**Pull request by jennetturkkan**
* Fixed margin between button and text. Had to move text into another component for that, but it doesn't affect anything
* Changed p color 
* Fixed href 
* Updated snapshot test
Screen Shot 2020-11-22 at 16 31 30

### CODE REVIEW

**Comment by jennetturkkan** [comment]
fixed issues you mentioned @louisrli. can you take a look?

Screen Shot 2020-11-22 at 19 26 26

File: src/components/SignUpForm/index.jsx [hunk 1]
```diff
@@ -124,25 +124,37 @@ export default function SignUpForm({ submit }) {
...
+                id="acceptTerms"
+                name="acceptTerms"
+                value="accepted"
+                aria-label="acceptTerms"
+                ref={register()}
+              />
+            </div>
+            <div>
+              <label htmlFor="acceptTerms">
+                I have read and agree with the{" "}
```
Comment (louisrli) [inline src/components/SignUpForm/index.jsx:141]: translatıon

Comment (louisrli) [reply src/components/SignUpForm/index.jsx:141]: is jamil working on translating this page

Comment (jennetturkkan) [reply src/components/SignUpForm/index.jsx:141]: yes, @Jamil-Bailony will add translation in a separate PR

**Review by louisrli — APPROVED** [review]

**Review by hiba-machfej — APPROVED** [review]

**Comment by jennetturkkan** [comment]
@Ammar-64

**Pull request merged**